src/physics/physics_factory.cpp

```diff
@@ -103,6 +103,7 @@ ::create_Physics(const Parameters::AllParameters                              *c
                 parameters_input->euler_param.angle_of_attack,
                 parameters_input->euler_param.side_slip_angle,
                 manufactured_solution_function,
+                false,
```

Comment:
Is this the bug?

Reply:
Swap the order of these two lines instead https://github.com/jbrillon/PHiLiP/blob/035653650946e14e14e9641ed49de6182b08d66d/src/physics/euler.h#L99-L100

Reply:
and these ones: https://github.com/jbrillon/PHiLiP/blob/035653650946e14e14e9641ed49de6182b08d66d/src/physics/navier_stokes.cpp#L33-L34

Reply:
Yes, it causes "has_nonzero_diffusion" to receive the value of "two_point_num_flux_type".

Reply:
I agree with your suggestion to swap the last two arguments. I'll make that change.